Crashlytics/Crashlytics/Settings/Models/FIRCLSApplicationIdentifierModel.m

```diff
@@ -132,7 +132,7 @@ - (void)computeInstanceIdentifier {
   // continue generating a less-correct value for now. One day, we should incorporate a hash of the
   // Info.plist and icon data.
 
-  _buildInstanceID = FIRCLSHashNSData([string dataUsingEncoding:NSUTF8StringEncoding]);
```

Comment:
Does FIRCLSHashNSData hash to the same value as FIRCLS256HashNSData?

If not, I think there's one thing to look into with this change. I forget what Build Instance ID is for in crashlytics, but if it's for reporting (eg. affects user counts), this could cause numbers to spike as the backend gets new instance IDs

Reply:
No they are different, even different by number of digits.